components/hubspot/hubspot.app.js

```diff
@@ -8,26 +9,28 @@ module.exports = {
       type: "string[]",
       label: "Lists",
       description: "Select the lists to watch for new contacts.",
-      async options(prevContext) {
-        const { offset = 0 } = prevContext;
+      async options({ page }) {
+        const count = 250;
+        const offset = page * count;
         const params = {
-          count: 250,
+          count,
           offset,
         };
         const results = await this.getLists(params);
         const options = results.map((result) => {
-          const { name: label, listId } = result;
+          const {
+            name: label,
+            listId,
+          } = result;
           return {
             label,
-            value: JSON.stringify({ label, value: listId }),
+            value: JSON.stringify({
```

Comment:
Why are you using `stringify` here and `parse` later? Am I missing something? I saw other occurrences of this in the `app`.

Reply:
It was originally so that the prop label could be emitted in the summary. Since neither `stages` or `forms` are being emitted in any component summaries, I removed the JSON.stringify from them. I left it in for `lists` though since the list name is emitted in new-contact-in-list.mjs.